sdk/src/main/java/com/karhoo/sdk/api/model/adyen/AdyenPaymentMethods.kt

```diff
@@ -0,0 +1,10 @@
+package com.karhoo.sdk.api.model.adyen
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class AdyenPaymentMethods(@SerializedName("groups") val groupAdyens: List<AdyenPaymentMethodsGroup>,
```

Comment:
@jsm20 @nurito-burrito 

https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/paymentMethods 

there seems to be two missing properties here - 

`oneClickPaymentMethods [RecurringDetail]`
`storedPaymentMethods [StoredPaymentMethod]`

were these intentionally left out? :)

Reply:
No. My bad as I auto generated from a response which didn't have those in. Will add them in the next NSDK endpoint PR.